src/core/useTrackBindingPlugin.js

```diff
@@ -82,8 +78,6 @@ export class TrackBindingPlugin {
         if (eventName) {
             delete dataset.eventName;
             callback(eventName, dataset, mergePagedefaults === "true");
-        } else {
-            callback(dataset, mergePagedefaults === "true");
         }
```

Comment:
`react-metrics` will throw an invariant error for missing `eventName` here
https://github.com/nfl/react-metrics/blob/master/src/core/createMetrics.js#L356-L359
so removing this line shouldn't have any effect. I don't exactly remember the reasoning for supporting w/o `eventName` here but i think i supported this in `useTrackBindingPlugin.js` in case i export this plugin and someone uses it standalone w/o eventName. I thought it's a breaking change but since we don't export this module to public so it will be a minor version bump as you mentioned.